plugins/crypto/ua_securitypolicy_filestore.c

```diff
@@ -178,13 +178,13 @@ writeCertificateAndPrivateKeyToFilestore(const UA_String storePath,
 
     /* Create the paths to the certificates and private key folders */
     char ownCertPathDir[PATH_MAX];
-    if(mp_snprintf(ownCertPathDir, PATH_MAX, "%.*s/%s", (int)storePath.length,
-                   (char *)storePath.data, "/own/certs") < 0)
+    if(mp_snprintf(ownCertPathDir, PATH_MAX, "%.*s%s", (int)storePath.length,
+                   (char *)storePath.data, "/ApplCerts/own/certs") < 0)
```

Comment:
Why this change?
Deep nested folders are bad.
And this subfolder thing is not documented.

Reply:
I will provide more documentation.
This function is triggered when the server certificate is updated using the UA_Server_updateCertificate function. It only updates the application instance certificate. The path to the PKI folder is stored in the SecurityPolicy_FilestoreContext, so you need to specify the necessary subfolder.